src/server/services/BruincastServices.js

```diff
@@ -37,6 +37,39 @@ class BruincastServices {
     return toBeReturned;
   }
 
+  // Given an array of casts sorted by date, return an array of objects with the casts grouped by week number
+  static groupCastsByWeek(casts) {
+    const docsByWeek = [];
```

Comment:
@chern Are you sure this cannot be done by the database? Can't you sort the results so they are grouped by week? See https://docs.mongodb.com/manual/reference/operator/aggregation/group/